### DISCUSSION TITLE
[Feature request] Enrich OS specific information on bug reports.

### DISCUSSION THREAD

**Discussion by tkatsoulas** [getsentry/sentry-native#942] [Ideas]
Hello guys, 

The current implementation to understand the host OS https://github.com/getsentry/sentry-native/blob/master/src/sentry_os.c provides poor information for any distro. Are there any plans to update them? For instance `/etc/os-releases` file is there for most* of the Linux distros and can provide detailed information about a host. The current mechanisms can be used as a fallback. 

Thanks in advance, 
Tasos.

**Comment by supervacuus** [comment]
That sounds like an excellent opportunity for a community contribution :-)

**Reply by tkatsoulas** [reply]
roger that
